client/testclient/cmd/lookup.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/coniks-sys/coniks-go/client"
+	"github.com/coniks-sys/coniks-go/keyserver/testutil"
+	p "github.com/coniks-sys/coniks-go/protocol"
+	"github.com/spf13/cobra"
+)
+
+var lookupCmd = &cobra.Command{
+	Use:   "lookup",
+	Short: "Lookup a name.",
+	Long:  `Lookup the key of some known contact.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		conf := loadConfigOrExit(cmd)
+		name := cmd.Flag("name").Value.String()
+		if len(name) == 0 {
+			cmd.Usage()
+			return
+		}
+		req, err := createLookupRequest(name)
+		if err != nil {
+			fmt.Println("Couldn'r create request!")
+			os.Exit(-1)
+		}
+		resp, err := testutil.NewTCPClient(req)
+		if err != nil {
+			fmt.Println("Error while retrieving repsonse: " + err.Error())
+			os.Exit(-1)
+		}
+		response, errCode := client.UnmarshalResponse(p.KeyLookupType,
+			resp)
+		res, ok := response.(*p.DirectoryProof)
+		if !ok {
+			fmt.Println("Got unexpected response from server!")
+			os.Exit(-1)
+		}
+		switch errCode {
+		case p.Success:
+			c := res.Verify(name, []byte(res.AP.Leaf.Value), 0, nil,
+				conf.SigningPubKey)
+			// verify auth. path:
+			if c != p.Passed || res.AP == nil ||
```

Comment:
I think we can store the latest STR to `client/config.toml` (with an explanation that that field is being used for testing only, the developers should actually store the latest STR in some persistent storages) and whenever the test client runs, it reads the latest STR from this config file. What do you think?